docs/non-normative/db-migration.md

```diff
@@ -22,6 +22,10 @@ updated to the stable database semantic conventions, they:
 - May drop the environment variable in their next major version and emit only
   the stable database conventions.
 
+> [!NOTE]
+> `OTEL_SEMCONV_STABILITY_OPT_IN` is only intended to be used when migrating
```

Comment:
would you mind adding the same to messaging and HTTP?

Reply:
added to HTTP, I didn't find a messaging migration guide?

Reply:
ah, right, it's still in PR, left a comment there - https://github.com/open-telemetry/semantic-conventions/pull/1198/files#r1920790089